test_add_armor fails

player_wizard = None, browser = <selenium.webdriver.safari.webdriver.WebDriver (session="E01B3898-7AB8-4C23-962D-9CF3E6F8364E")>

    def test_add_armor(player_wizard, browser): # noqa
        """As a player, I can add an armor."""
        print('As a player, I can add an armor.')

        armor_add = armor.ArmorAddModal(browser)
        armor_table = armor.ArmorTable(browser)
        tabs = Tabs(browser)
        tabs.equipment.click()

        stub = ArmorFactory.stub()

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
                (By.ID, armor_table.add_id)
            )
        )

        armor_table.add.click()
        armor_add.name = stub.name
        armor_add.type_ = stub.type_
        armor_add.magical_modifier = stub.magical_modifier
        armor_add.price = stub.price
        armor_add.currency_denomination = stub.currency_denomination
        armor_add.weight = stub.weight
        armor_add.armor_class = stub.armor_class
        armor_add.stealth = stub.stealth
        armor_add.stealth.send_keys(Keys.TAB)
        armor_add.don.click()
        armor_add.description = stub.description

        assert armor_add.name.get_attribute('value').strip() == stub.name
        assert armor_add.type_.get_attribute('value').strip() == stub.type_
        assert int(armor_add.magical_modifier.get_attribute('value').strip()) == stub.magical_modifier
        assert int(armor_add.price.get_attribute('value').strip()) == stub.price
        assert armor_add.currency_denomination.get_attribute('value').strip() == stub.currency_denomination
        assert int(armor_add.weight.get_attribute('value').strip()) == stub.weight
        assert int(armor_add.armor_class.get_attribute('value').strip()) == stub.armor_class
        assert armor_add.stealth.get_attribute('value').strip() == stub.stealth
        assert 'active' in armor_add.don.get_attribute('class').strip()
        assert armor_add.description.get_attribute('value').strip() == stub.description

        armor_add.add.click()

        row = ut.get_table_row(armor_table, 'table', 1)

>       assert row.armor.strip() == '{}  + {}'.format(
            stub.name, stub.magical_modifier
        )
E       AssertionError: assert 'Armor_tbpmcu...          + 2' == 'Armor_tbpmcuc...lJFInQxb  + 2'
E         - Armor_tbpmcucmHacqeCxNqPCfurhlVQTghwyJlJFInQxb
E         ?                                               ^^
E         + Armor_tbpmcucmHacqeCxNqPCfurhlVQTghwyJlJFInQxb  + 2
E         ?                                               ^^^^^
E         -               + 2

--- a/tests/core/player_tools/test_equipment.py
+++ b/tests/core/player_tools/test_equipment.py
@@ -791,7 +791,9 @@ def test_add_armor(player_wizard, browser): # noqa
 
     row = ut.get_table_row(armor_table, 'table', 1)
 
-    assert row.armor.strip() == '{}  + {}'.format(
+    armor_name_label = ' '.join([string.strip() for string in row.armor.split()])
+
+    assert armor_name_label == '{} + {}'.format(
         stub.name, stub.magical_modifier
     )
     assert int(row.armor_class.strip()) == stub.armor_class

